solverdummy/solverdummy.py

```diff
@@ -33,7 +34,8 @@
 solver_process_index = 0
 solver_process_size = 1
 
-interface = precice.Interface(participant_name, configuration_file_name, solver_process_index, solver_process_size)
+interface = precice.Interface(participant_name, configuration_file_name,
+                              solver_process_index, solver_process_size)
 
```

Comment:
```suggestion
read_data_name, write_data_name = None, None
```
This line can go even further below but I was not able to add a suggestion where there was no code change in the review.